src/App/Init.php

```diff
@@ -627,6 +634,12 @@
 $di->params[Ushahidi\App\Validator\User\Reset::class] = [
 	'repo'    => $di->lazyGet('repository.user')
 ];
+$di->params[Ushahidi\App\Validator\User\Setting\Update::class] = [
+	'user_repo'    => $di->lazyGet('repository.user')
+];
+$di->params[Ushahidi\App\Validator\Contact\Update::class] = [
```

Comment:
Why did this change in this PR?

Reply:
It was changed because of fiddling with white space in editors.

Reply:
Our white space is wrong everywhere.